internal/xds/bootstrap/bootstrap.yaml.tpl

```diff
@@ -113,26 +113,40 @@ static_resources:
                   envoy.filters.http.compression:
                     "@type": type.googleapis.com/envoy.extensions.filters.http.compressor.v3.CompressorPerRoute
                     overrides:
+                      {{- if .PrometheusCompressionRemoveAcceptEncodingHeader }}
```

Comment:
thinking this loud, should we move 19001 to dynamic listener
cc @envoyproxy/gateway-maintainers